hooks/pre/30-upgrade.rb

```diff
@@ -45,12 +45,12 @@ def remove_gutterball
 def migrate_pulp
   # Fix pid if neccessary
   if Kafo::Helpers.execute("grep -qe '7.[[:digit:]]' /etc/redhat-release")
-    Kafo::Helpers.execute("sed -i -e 's?/var/run/mongodb/mongodb.pid?/var/run/mongodb/mongod.pid?g' /etc/mongod.conf")
+    Kafo::Helpers.execute("sed -i -e 's?/var/opt/rh/rh-mongodb34/run/mongodb/mongod.pid?/var/opt/rh/rh-mongodb34/run/mongodb/mongod.pid?g' /etc/opt/rh/rh-mongodb34/mongod.conf")
```

Comment:
Am I missing something or is this a noop? I think this whole block can be deleted since can assume the pid is just fine in the SCL.

Reply:
I agree, I think this was an old work around we shouldnt need anymore.